src/main/kotlin/ui/ConfigurationEditor.kt

```diff
@@ -400,11 +438,13 @@ class ConfigurationEditor(props: ConfigurationEditorProps) :
 
                 val format = props.outputFormat
                 delay(100)
-                val availableFeatures = Feature.values().filter {
+                var availableFeatures = Feature.values().filter {
                     it.isAvailable.invoke(project) &&
                             format.availableFeaturesForGeneration.contains(it)
                 }
-                // TODO: Add UI for users to select features
+
+                availableFeatures = removeUncheckedFeatures(availableFeatures)
```

Comment:
```kotlin
                val availableFeatures = Feature.values().filter {
                     it.isAvailable.invoke(project) &&
                             format.availableFeaturesForGeneration.contains(it)
                 }.let(::removeUncheckedFeatures)
```
OR
```kotlin
                val availableFeatures = Feature.values().filter {
                     it.isAvailable.invoke(project) &&
                             format.availableFeaturesForGeneration.contains(it)
                 }.let { removeUncheckedFeatures(it) }
```
To avoid changing to it `var`